client/src/components/Login.jsx

```diff
@@ -58,7 +61,7 @@ const Login = props => {
           <Button variant="login" className={classes.Button} onClick={onClick}>
             {t('login.buttonText')}
           </Button>
-          <TranslationsMenu />
+          <TranslationsMenu /> hi
```

Comment:
Let's remove this

Reply:
Will do!

Reply:
<img width="1676" alt="Screen Shot 2020-04-20 at 3 48 58 PM" src="https://user-images.githubusercontent.com/47477912/79793515-b52b6900-831e-11ea-8584-2c5865a783b0.png">
This is what it looks like full screened (desktop). I'll try to move the button to the center (vertically) like you suggested!